cfme/tests/automate/test_domain.py

```diff
@@ -393,3 +397,59 @@ def test_copy_to_domain(domain):
     method = domain.browser.create_view(MethodCopyView)
     method.flash.wait_displayed()
     method.flash.assert_message("Copy selected Automate Method was saved")
+
+
+@pytest.fixture(scope="function")
+def new_user(request, appliance):
+    """This fixture creates custom user with tenant attached"""
+    tenant = _tenants(request, appliance)
+    role = appliance.rest_api.collections.roles.get(name="EvmRole-super_administrator")
+    group = _groups(request, appliance, role, tenant=tenant)
+    user, user_data = _users(request, appliance, group=group.description)
+    yield appliance.collections.users.instantiate(
```

Comment:
**Suggestion:**
You can use `return` here instead of `yield`, as post operation is being done by the `create skeleton` which has tear down operation.